docs/site/pages/getting-started.mdx

```diff
@@ -4,27 +4,66 @@ title: 'Getting Started'
 
 # Getting Started
 
-Getting started with Player is simple. Below we have guides per platform on how to integrate Player into your platform of choice. 
-
-## React 
+Getting started with Player is simple.
 
 ### Install Dependencies
 
-The first dependency you'll need to pull in in the React Player itself. You can do this by running
+The first dependency you'll need to pull in is the Player itself. Additionally, you'll need an assets plugin to define any UI -- we'll use the reference assets as an example.
+
+<PlatformTabs>
+  <react>
+
+You can do this by running
 
 ```bash
 yarn add @player-ui/react
+yarn add @player-ui/reference-assets-plugin-react
```

Comment:
Probably out of scope for this PR but we should probably determine which package manager we write our JS docs for. Yarn seems fine for now but when we move to pnpm should we also move these examples?